### DISCUSSION TITLE
Chart title not appearing on SaveToPng (WinUI3)

### DISCUSSION THREAD

**Discussion by gd-etiltd** [Live-Charts/LiveCharts2#1138] [Q&A] [answered]
I have a Cartesian line chart in WinUI3 that works fine in the application. I have added a 'Save image' button with the following code:

```
var title =
  new LabelVisual
  {
    Text = "My Chart Title",
    TextSize = 20,
  };

SKCartesianChart skChart =
  new (this.LvcGraph)
    {
      Width = 1920,
      Height = 1280,
      LegendPosition = LiveChartsCore.Measure.LegendPosition.Right,
      Title = title,
    };

skChart.SaveImage("C:\\Temp\\CartesianImageFromControl.png");
```

This saves the image of the chart fine - but there is no title showing on it. Obviously I'm doing something wrong - but there seems to be no documentation relating to Chart Titles. Can somebody point out my mistake? Thanks!

**Comment by gd-etiltd** [comment, marked as answer]
Found the resolution in one of the recent releases:

[https://github.com/beto-rodriguez/LiveCharts2/releases](https://github.com/beto-rodriguez/LiveCharts2/releases
)

Added a value to the paint field and its now appearing as expected.

**Reply by beto-rodriguez** [reply]
Glad you found a solution!
